Update to Flood control

TsunayoshiSawada opened the pull request:
Flood control now mutes a user instead of banning.

architdate commented:
why create #1 and #2 both? don't they serve the same purpose?

anandvfcvijay commented:
Sorry it was an accidental mistake due to network error

On Wed, Aug 14, 2019, 11:09 AM Archit Date <notifications@github.com> wrote:

> why create #1 <https://github.com/AnimeKaizoku/SaitamaRobot/pull/1> and #2
> <https://github.com/AnimeKaizoku/SaitamaRobot/pull/2> both? don't they
> serve the same purpose?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AnimeKaizoku/SaitamaRobot/pull/2?email_source=notifications&email_token=ALXCMCXDU53JYQU7UY65YB3QEOLA7A5CNFSM4ILQF4DKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD4HXZRY#issuecomment-521108679>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALXCMCTFENJSA3QVLY4WPMLQEOLA7ANCNFSM4ILQF4DA>
> .
>

anandvfcvijay commented:
One is the requirement  [googletrans]

architdate commented:
Yeah that can be pip installed, so not an issue there. If possible can you remove the commented out code. Apart from that it looks fine and ready to merge for me

anandvfcvijay commented:
Ok

On Wed, Aug 14, 2019, 11:18 AM Archit Date <notifications@github.com> wrote:

> Yeah that can be pip installed, so not an issue there. If possible can you
> remove the commented out code. Apart from that it looks fine and ready to
> merge for me
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AnimeKaizoku/SaitamaRobot/pull/2?email_source=notifications&email_token=ALXCMCXO33SN5RTOF42IHE3QEOMCLA5CNFSM4ILQF4DKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD4HYGXY#issuecomment-521110367>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALXCMCXQ63J7LLNY7KYEZJTQEOMCLANCNFSM4ILQF4DA>
> .
>

TsunayoshiSawada commented:
Anand - Also include an edit to requirements.txt and we will collectively merge all of this.

anandvfcvijay commented:
Just add googletrans as requirement for the Translator

On Wed, Aug 14, 2019, 11:33 AM Sawada Tsunayoshi <notifications@github.com>
wrote:

> Anand - Also include an edit to requirements.txt and we will collectively
> merge all of this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AnimeKaizoku/SaitamaRobot/pull/2?email_source=notifications&email_token=ALXCMCV6G3TVIIBO2UXFUHLQEON2LA5CNFSM4ILQF4DKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD4HY6BY#issuecomment-521113351>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALXCMCTAI6OSU4VRMDRM2UDQEON2LANCNFSM4ILQF4DA>
> .
>

architdate commented:
When you are deleting the comments in the above file, just also add googletrans to requirements.txt like you did in #1 once that's done, i'll squash and merge. Thanks for ur contribution

TsunayoshiSawada committed:
Change flood ban to mute

Note: I'm not adept in messing around functions like this, this is merely a simple edit and is possibly unstable or unreliable. 
A review is requested by someone who knows this better.

TsunayoshiSawada committed:
Update response on flood mute

Another possible experimental edit that may need review.

TsunayoshiSawada committed:
Quick fix for missing parsing on flood mention

TsunayoshiSawada committed:
Desuuuuuuuuuuuuu

Nothing to see on this one
